File: LeetCode/Length_of_Last_Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = s.strip()
        i = len(s)-1
        length = 0
        while i >= 0:
            if str.isalnum(s[i]):
                length +=1
                i -=1
            else:
                break
        return length

File: LeetCode/Length_of_Last_Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int length, index;
        index = s.size() - 1;
        length = 0;
        
        while (isspace(s[index])) {
            index --;
        }
        while (isalnum(s[index])) {
            length++;
            index--;
        }
    return length;
    }
};
